Assets/Scenes/SampleScene.unity

```diff
@@ -631,6 +635,79 @@ RectTransform:
   m_AnchoredPosition: {x: 0, y: 0}
   m_SizeDelta: {x: 0, y: 0}
   m_Pivot: {x: 0, y: 0}
+--- !u!1001 &2617945920427387056
+PrefabInstance:
+  m_ObjectHideFlags: 0
+  serializedVersion: 2
+  m_Modification:
+    m_TransformParent: {fileID: 0}
+    m_Modifications:
+    - target: {fileID: 2617945921448679910, guid: 309730eacf554f04e991349c77ea6ee5, type: 3}
+      propertyPath: m_Name
+      value: Win Condition
```

Comment:
The Level End Trigger prefab instance in `SampleScene.unity` should be renamed from Win Condition to Level End Trigger.